package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "dotenv-parse-variables",
   "description": "Parse dotenv files for Boolean, Array, and Number variable types, built for CrocodileJS",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
@madoke can you please undo the version bump and do not touch package.json file?  I handle this using `np` tool.  thank you

Reply:
sure, will do shortly

Reply:
done 👍